processor/src/main/java/org/mapstruct/ap/spi/ImmutablesAccessorNamingStrategy.java

```diff
@@ -19,6 +20,17 @@
 @Experimental("The Immutables accessor naming strategy might change in a subsequent release")
 public class ImmutablesAccessorNamingStrategy extends DefaultAccessorNamingStrategy {
 
+    @Override
+    public boolean isGetterMethod(ExecutableElement method) {
+        return super.isGetterMethod( method ) || isFluentGetter( method );
+    }
+
+    private boolean isFluentGetter(ExecutableElement method) {
+        return method.getParameters().isEmpty() &&
+            method.getReturnType().getKind() != TypeKind.VOID &&
+            !JAVA_JAVAX_PACKAGE.matcher( method.getEnclosingElement().asType().toString() ).matches();
+    }
```

Comment:
I don't think that we should just make this like this. We should look at the Immutables `@Styles`, see https://immutables.github.io/style.html and determine based on that.

Otherwise, this just applies to everything.

Additionally, I really do not understand how this even works, the `getElementName` is not overridden, so I don't think that this works properly